### ISSUE TITLE
apache.status system test is failing

### ISSUE THREAD

**Issue by ycombinator** [elastic/elastic-package#374]
The system test for the `apache.status` data stream fails CI on `master` with this error:

```
one or more errors found in documents stored in metrics-apache.status-ep data stream: [0] field "apache.status.hostname" is undefined
[1] field "apache.status.total_kbytes" is undefined
```

Example CI failure: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Felastic-package/detail/master/115/tests/

**Comment by ycombinator** [comment]
Doing some digging, I wonder if this has something to do with Agent no longer starting Metricbeat with `-E management.mode=x-pack-fleet` from 7.14 onwards? cc: @mtojek

**Comment by ycombinator** [comment]
I'm skipping this system test for now, to unblock other PRs: https://github.com/elastic/elastic-package/pull/375. Once we figure out why this test is suddenly failing and resolve the issue, the test should be un-skipped.

**Comment by ycombinator** [comment]
I think I found the issue. This PR was recently merged into the `7.x` branch: https://github.com/elastic/beats/pull/26012. As part of that PR, this line was removed: https://github.com/elastic/beats/pull/26012/files#diff-7a841954bb3e58e0c8cd54e0ece78075a30bd6f19f5a45a7c8e314c549f6b6a1L6.

**Comment by ycombinator** [comment]
I believe the fix for this is in either the Agent code or the Metricbeat `apache/status` metricset code. Either way, I've created https://github.com/elastic/beats/issues/26120 to track the fix.

**Comment by mtojek** [comment]
Looks like the fix is merged, so resolving this one.
